<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1124</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="txt">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>471</width>
      <height>361</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Type Text...</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>70</y>
      <width>471</width>
      <height>361</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="input_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Input:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_2">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>30</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="translate_btn">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>450</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: #fff;
	background-color: rgb(76, 88, 255);
	border: 3px solid rgb(76, 88, 255);
	border-radius: 5px;
}
QPushButton:hover {
	background-color: blue;
	border-color: blue;
}</string>
    </property>
    <property name="text">
     <string>Translate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter Language:</string>
    </property>
   </widget>
   <widget class="QLabel" name="out_label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Out:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter Language:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_btn">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>90</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: #fff;
	background-color: rgb(76, 88, 255);
	border: 3px solid rgb(76, 88, 255);
	border-radius: 5px;
}
QPushButton:hover {
	background-color: blue;
	border-color: blue;
}</string>
    </property>
    <property name="text">
     <string>Clear All</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1124</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="help_menu">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="help_action"/>
    <addaction name="about_action"/>
    <addaction name="separator"/>
    <addaction name="feedback_action"/>
   </widget>
   <widget class="QMenu" name="options_menu">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="exit_action"/>
   </widget>
   <addaction name="options_menu"/>
   <addaction name="help_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="help_action">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="about_action">
   <property name="text">
    <string>About App</string>
   </property>
  </action>
  <action name="feedback_action">
   <property name="text">
    <string>Send Feedback</string>
   </property>
  </action>
  <action name="exit_action">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
